### ISSUE TITLE
Example code generateClaimCodeForOrganization doesn't return claim code.

### ISSUE THREAD

**Issue by steven7gn** [particle-iot/ios-app-example-pod#6]
Hi- all, am stuck and could use some help. We're hoping to get an ios App up and running for some beta customer setups in the next several weeks. Unfortunately it doesn't seem to work to add a device into an organization. The SparkSetupExample-Swift project looks ideal for some minor modification for us. 

So-- I've set up a SparkSetupExample-Swift project successfully in Xcode and I'm able to successfully add a device when using my developer account. (Required commenting out 2 lines of code that were effectively no-ops). I've set up an organization dashboard and initial experimentation shows it is working ok (I set up a C# test project and have been accessing the organization with the .net webClient with reasonable success for organization cloud calls although I wasn't able to get claims to work at all).

In SparkSetupExample-Swift, when I add in productSlug and organizationSlug and set organization to true, I get a failure during the soft setup process. The debug window in XCODE shows that the code was executing generateClaimCodeForOrganization https://api.particle.io/v1/orga//products//device_claims Failed with a status code of 404, error = "Organization not found for user.". I'm set up as the user in the organization.

Can anyone help me to debug a device_claims failed? Any chance this might be an issue at the cloud API level? I wasn't able to get cloud API https:// put commands to work for device claims......

**Comment by idokleinman** [comment]
Hi Steven,
seems like the URL line https://api.particle.io/v1/orga//products//device_claims is invalid - having two // and the name of the org and product missing.. Did you trace this issue or I am missing something here?

**Comment by steven7gn** [comment]
Hi -  thanks for replying back.   The generateClaimCodeForOrganization line was mangled when I submitted the note and I didn't notice it upon submitting it-- sorry about that.   That line was set it up as https://api.particle.io/v1/brayden-automation/web-wabbit5-v100/device_claims.   I logged in as the developer when running the experiment.   Just in case,    I've also tried that line as https://api.particle.io/v1/brayden-automation/web-wabbit5/device_claims as indicated in the documentation (there's a discrepancy in what's specified as a device name as to whether the firmware revision is added to the name or not).
